clustered/common/src/main/java/org/ehcache/clustered/common/internal/util/ExtensionInfoSkeleton.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.common.internal.util;
+
+import com.tc.productinfo.ExtensionInfo;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.jar.Attributes;
+import java.util.jar.JarFile;
+
+/**
+ * @author Mathieu Carbou
+ */
+public abstract class ExtensionInfoSkeleton implements ExtensionInfo {
```

Comment:
Not sure why this class is here and not in platform... it doesn't seem to be Ehcache specific.

Reply:
It is... In platform is is implemented another way to retrieve the details from the jar files.
Also, I have no issue in extending the one in platform. But it would introduce a dependency over DC in Ehcache (server-side). Do we want that ?